src/Db/Odbc/VerticaOdbcAbstract.php

```diff
@@ -251,7 +250,7 @@ public function update($tableName, array $parameters, $where)
         foreach ($parameters as $column => $value) {
             $sql .= $column . " = {$this->quote($value)},";
         }
-        $sql = rtrim($sql, ',') . ' WHERE ' . (!empty($where) ? $where : '1');
+        $sql = rtrim($sql, ', ') . ' WHERE ' . (!empty($where) ? $where : '1');
```

Comment:
For the insert method your fix does makes sense. But not here. See 2 lines above: there is no space after the comma.
Please adjust accordingly.
Thank you.